include/exadg/aero_acoustic/driver.cpp

```diff
@@ -135,26 +123,49 @@ template<int dim, typename Number>
 void
 Driver<dim, Number>::solve()
 {
-  set_start_time();
+  std::pair<bool, dealii::Timer> timer = std::make_pair(false, dealii::Timer());
 
-  synchronize_time_step_size();
+  set_start_time();
 
   AssertThrow(std::abs(application->fluid->get_parameters().end_time -
                        application->acoustic->get_parameters().end_time) < 1.0e-12,
               dealii::ExcMessage("Acoustic and fluid simulation need the same end time."));
 
   while(not acoustic->time_integrator->finished())
   {
+    if(timer.first == false && acoustic->time_integrator->started())
+    {
+      timer.first = true;
+      timer.second.restart();
+    }
+
+    // To check if acoustics starts during the following sub-stepping sweep we
+    // can not simply check acoustic->time_integrator->started(). Instead we
+    // compute this information as follows:
+    const bool acoustic_starts_during_sub_stepping =
+      fluid->time_integrator->get_next_time() + fluid->time_integrator->get_time_step_size() >
+      application->acoustic->get_parameters().start_time;
```

Comment:
I think we can improve the naming of `acoustic_starts_during_sub_stepping` and `acoustics_might_start`. The "sub-stepping" nomenclature should not be necessary on this level. I would suggest

- `acoustic_starts_during_present_timestep`
- `acoustic_might_start_during_next_timestep`